### ISSUE TITLE
Organs: Could CCF API include information for ASCT-B and icons? or even more?

### ISSUE THREAD

**Issue by mccalluc** [hubmapconsortium/portal-ui#2944]
Since CCF reference organs, ASCT-B, and the icons are all coming from IU, it would be great if your API could provide all the information to link them together. When this is available, the corresponding fields could be removed from `descriptions.yaml` in portal-ui, and updates will be smoother in the future.

(If you'd like to be a one-stop-shop and also provide the corresponding terms for the search-api, and for azimuth, we could dispense with `descriptions.yaml` entirely.)

**Comment by bherr2** [comment]
Yes, we've had calls to add icon links to reference organs. We have a new HRA-CCF release coming out in December we can work to add that piece for sure.
